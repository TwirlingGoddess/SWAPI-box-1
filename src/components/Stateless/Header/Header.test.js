// import React from 'react';
// import { shallow, mount } from 'enzyme';
// import Header from './Header';

// let mockProps = {
//   makeApiCall: jest.fn(),
//   favoritesLength: 1,
//   displayFavorites: jest.fn()
// };


// describe ('', () => {
// let header;
//   beforeEach(() => {
//     header = shallow(<Header {...mockProps}/>);
//   });

//   it('matches snapshot', () => {
//     expect(header).toMatchSnapshot();
//   });


// });