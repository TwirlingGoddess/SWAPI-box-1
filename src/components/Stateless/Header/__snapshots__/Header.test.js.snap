// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    displayFavorites={[Function]}
    favoritesLength={1}
    makeApiCall={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="logo"
>
           
          <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    to="/"
          />
</div>,
        <Buttons
          apiCall={undefined}
          displayFavorites={[Function]}
          favoritesLength={1}
/>,
      ],
      "className": "Header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            " ",
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              to="/"
/>,
          ],
          "className": "logo",
        },
        "ref": null,
        "rendered": Array [
          " ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "to": "/",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "apiCall": undefined,
          "displayFavorites": [Function],
          "favoritesLength": 1,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="logo"
>
             
            <NavLink
                        activeClassName="active"
                        ariaCurrent="true"
                        to="/"
            />
</div>,
          <Buttons
            apiCall={undefined}
            displayFavorites={[Function]}
            favoritesLength={1}
/>,
        ],
        "className": "Header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              " ",
              <NavLink
                activeClassName="active"
                ariaCurrent="true"
                to="/"
/>,
            ],
            "className": "logo",
          },
          "ref": null,
          "rendered": Array [
            " ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "to": "/",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "apiCall": undefined,
            "displayFavorites": [Function],
            "favoritesLength": 1,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
